### PR TITLE
Autofill harvest year to current year

### PR DESCRIPTION
**Pull request by badnames**
The aim of this commit is to prefill the year input field in the seed creation dialog with the current year (As requested in #35).
This is accomplished by adding a new defaultValue prop to SimpleFormInput.

## Basics

- [x] Details of what you changed are in commit messages
      (first line should have `module: short statement` syntax)
- [ ] References to issues, e.g. `close #X`, are in the commit messages.
- [x] The buildserver is happy.
- [x] The PR is rebased with current master.

## Checklist

- [x] I added unit tests for my code
- [x] I fully described what my PR does in the documentation
      (not in the PR description)
- [x] I fixed all affected documentation
- [x] I fixed all affected decisions
- [x] I added code comments, logging, and assertions as appropriate
- [x] I mentioned [every code](/.reuse/dep5) not directly written by me in [reuse syntax](https://reuse.software/)

## Review

- [x] Documentation is conforming to [our Documentation Guidelines](/doc/documentation.md)
- [x] Examples are well chosen and understandable
- [x] Code is conforming to our Coding Guidelines
- [x] Code is consistent to [our Design Decisions](/doc/decisions)

### CODE REVIEW

File: frontend/src/components/Form/SimpleFormInput.tsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import { FieldValues, Path, UseFormRegister } from 'react-hook-form';
 
-import { HTMLInputTypeAttribute } from 'react';
+import { useState, HTMLInputTypeAttribute } from 'react';
```
Comment (buenaflor) [inline frontend/src/components/Form/SimpleFormInput.tsx:3]: is `useState` needed?

**Review by buenaflor — APPROVED** [review]
left 1 comment but lgtm

**Comment by markus2330** [comment]
Great to see the continuation of the work for the seed milestone. We are so much looking forward to it! :rocket:

@buenaflor thank you so much for reviewing! :heart:

Is this rebased to current master? The deployment does seem not work? (According to Jenkins log and also https://pr.permaplant.net/seeds/new doesn't seem to have the auto-fill feature?)

File: frontend/src/components/Form/SimpleFormInput.tsx (hunk 1 above)
Comment (badnames) [reply frontend/src/components/Form/SimpleFormInput.tsx:3]: No, its not. I was initially going to use it, but I forgot to remove it in the end.

**Comment by badnames** [comment]
> Is this rebased to current master? The deployment does seem not work? (According to Jenkins log and also 
> https://pr.permaplant.net/seeds/new doesn't seem to have the auto-fill feature?)

 I think so.... 
GitHub shows a green tick mark, so I assumed that it is ready to merge.

File: frontend/src/components/Form/SimpleFormInput.tsx (hunk 1 above)
Comment (buenaflor) [reply frontend/src/components/Form/SimpleFormInput.tsx]: Hm surprised this goes through the pre-commit hook. We have a linter setup that should execute when committing, I'll have to take a look at it.

**Pull request merged**